src/runtime/runtime-simd.cc

```diff
@@ -301,8 +301,13 @@ SIMD128_UNARY_FUNCTIONS(DECLARE_SIMD_UNARY_FUNCTION)
 
 
 template<typename T1, typename T2>
-inline void BitsTo(T1 s, T2* t) {
-  memcpy(t, &s, sizeof(T2));
+inline void BitsTo(T1 *s, T2* t) {
+  memcpy(t, s, 1);
+}
```

Comment:
implement bits copy, when the function SIMD.**.from.**Bits is invoked, this function will worked, the order function is abandoned. I will revise this function to reduce the iterations times.